akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowMapAsyncUnorderedSpec.scala

```diff
@@ -52,6 +52,27 @@ class FlowMapAsyncUnorderedSpec extends StreamSpec {
       c.expectComplete()
     }
 
+    "complete without requiring further demand" in assertAllStagesStopped {
+      import system.dispatcher
+      Source
+        .single(1)
+        .mapAsyncUnordered(1)(v => Future { Thread.sleep(20); v })
+        .runWith(TestSink.probe[Int])
+        .request(1)
+        .expectNext(1)
```

Comment:
```suggestion
        .requestNext(1)
```

same in the other tests